src/service.ts

```diff
@@ -137,6 +137,19 @@ export class DebugService implements IDebugger {
     }
   }
 
+  /**
+   * Makes the current thread step out a function / method if possible.
+   */
+  async stepOut(): Promise<void> {
+    try {
+      await this.session.sendRequest('stepOut', {
+        threadId: this.currentThread()
+      });
+    } catch (err) {
```

Comment:
Do we want this `catch`? Or should the promise actually fail if the request fails?

Reply:
Eventually we'll want to remove the `try catch` and let the promise fail, so we can handle the error properly (showing an error message to the user for example).

Reply:
Thanks @afshin for the suggestion, opened https://github.com/jupyterlab/debugger/issues/147 to keep track of it.